functions/New-DbaAgentJob.ps1

```diff
@@ -0,0 +1,476 @@
+                    $Job.OperatorToEmail = $EmailOperatorName
+                }
+                else 
+                {
+                    Stop-Function -Message ("The e-mail operator name '$($EmailOperatorName)' does not exist on instance '$($SqlServer)'. Exiting..") -Silent $Silent -Target $JobName
+                    return
+                }
+            }
+            else 
+            {
+                Write-Message -Message "Invalid combination of e-mail operator name '$($EmailOperatorName)' and email level $($EmailLevel). Not setting the notification." -Warning -Silent $Silent
+            }
+        }
+        
+
+        if($NetsendOperatorName.Length -ge 1)
+        {
+            if($NetsendLevel -ge 1)
+            {
+                # Check if the operator name is present
+                if(($Server.JobServer.Operators).Name -contains $NetsendOperatorName)
+                {
+                    Write-Message -Message "Setting job netsend level" -Level 5 -Silent $Silent
+                    $Job.NetSendLevel = $NetsendLevel
+
+                    Write-Message -Message "Setting job netsend operator" -Level 5 -Silent $Silent
+                    $Job.OperatorToNetSend = $NetsendOperatorName
+                }
+                else 
+                {
+                    Stop-Function -Message ("The netsend operator name '$($NetsendOperatorName)' does not exist on instance '$($SqlServer)'. Exiting..") -Silent $Silent -Target $JobName
+                    return
+                }   
+            }  
+            else 
+            {
+                Write-Message -Message "Invalid combination of netsend operator name '$($NetsendOperatorName)' and netsend level $($NetsendLevel). Not setting the notification." -Warning -Silent $Silent
+            }        
+        }
+
+        if($PageOperatorName.Length -ge 1)
+        {
+            if($PageLevel -ge 1)
+            {
+                # Check if the operator name is present
+                if(($Server.JobServer.Operators).Name -contains $PageOperatorName)
+                {
+                    Write-Message -Message "Setting job pager level" -Level 5 -Silent $Silent
+                    $Job.PageLevel = $PageLevel
+
+                    Write-Message -Message "Setting job pager operator" -Level 5 -Silent $Silent
+                    $Job.OperatorToPage = $PageOperatorName
+                }
+                else 
+                {
+                    Stop-Function -Message ("The page operator name '$($PageOperatorName)' does not exist on instance '$($SqlServer)'. Exiting..") -Silent $Silent -Target $JobName
+                    return
+                }
+            }
+            else 
+            {
+                Write-Message -Message "Invalid combination of page operator name '$($PageOperatorName)' and page level $($PageLevel). Not setting the notification." -Warning -Silent $Silent
+            }
+        }
+
+        if($DeleteLevel -ge 0)
```

Comment:
if we remove the '' as default for the parameter, and there's a validateset,
and we add default {0} where the strings are switched to int in the BEGIN{}, 
these if () are redundant